maxim/logger/portkey/client.py

```
@@ -0,0 +1,37 @@
+from typing import Any
+
+try:
+    from portkey_ai.api_resources.client import Portkey, AsyncPortkey
+except ImportError as e:
+    raise ImportError(
+        "The portkey-ai package is required. Please install it using pip: `pip install portkey-ai` or `uv add portkey-ai`"
+    ) from e
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)